yarn.lock

```diff
@@ -21,14 +21,7 @@
   dependencies:
```

Comment:
Pushed by accident?

Reply:
Hmm no.. I did `yarn install` a couple of times because I was a bit confused about `emotion` still being there (it's a dependency of storybook), that must've changed some things in yarn.lock, but I think its ok?